products/bridge/smart-contracts/contracts/Relayer.sol

```diff
@@ -107,9 +110,46 @@ contract Relayer {
             call,
             readonly,
             callback,
-            nonces[msg.sender]
+            _nonces[msg.sender]
         );
-        return ++nonces[msg.sender];
+        return ++_nonces[msg.sender];
+    }
+
+    // TODO: remove later, used for testing
+    function warmup() external {
+        ++_refund[msg.sender];
+    }
+
+    function topUpGas(address target) external payable {
+        _gas[target] += msg.value;
```

Comment:
The variable name `_gas` is somewhat misleading as it stores the wei amount and not the gas amount (wei = gas * price)

Reply:
Completely agree with this. what would you think is a good name for this?

Reply:
I would call it something like `_feeDeposit`, as it is actually to cover the transaction fees the validators spend on dispatching the cross-chain calls to the twin